modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -203,8 +202,6 @@
 				-webkit-text-stroke-color var(--n-tabs-title-transition ),
 				transform var(--n-tabs-title-transition);
 
-			@include button-focus-outline;
```

Comment:
We prefer to use the default styling inside, which was overridden by the all: unset styling.